_posts/2023-01-26-rust-32.0.0.md

```diff
@@ -0,0 +1,189 @@
+---
+layout: post
+title: "January 2023 Rust Apache Arrow 32.0.0 Highlights"
+date: "2023-01-26 00:00:00 +0000"
+author: pmc
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+# Introduction
+
+With the recent release of version 32.0.0 of the Rust implementation of [Apache Arrow](https://arrow.apache.org/), it seemed timely to highlight some exciting things the community has been working on since the [last update](https://arrow.apache.org/blog/2022/06/16/rust-16.0.0/).
+
+The most recent set of changes can be found in the [CHANGELOG](https://github.com/apache/arrow-rs/blob/master/CHANGELOG.md), with the full historical list available [here](https://github.com/apache/arrow-rs/blob/master/CHANGELOG-old.md).
+
+# Arrow
+
+_TODO: Some pre-amble_
+
+**Major Highlights**
```

Comment:
I wasn't sure whether to include Run-end encoded arrays as they're very bleeding edge :sweat_smile:

Reply:
Maybe we can put it in next blog?